esphome/components/light/__init__.py

```diff
@@ -145,6 +151,32 @@ async def setup_light_core_(light_var, output_var, config):
 
     cg.add(light_var.set_restore_mode(config[CONF_RESTORE_MODE]))
 
+    if (initial_state_config := config.get(CONF_INITIAL_STATE)) is not None:
+        initial_state = cg.StructInitializer(
+            LightStateRTCState,
+            (
+                "color_mode",
+                initial_state_config.get(CONF_COLOR_MODE, ColorMode.UNKNOWN),
+            ),
+            ("state", initial_state_config.get(CONF_STATE, False)),
+            ("brightness", initial_state_config.get("brightness", 1.0)),
```

Comment:
Curious, why not import the constants for these and use them, instead? Like this:
```suggestion
            ("brightness", initial_state_config.get(CONF_BRIGHTNESS, 1.0)),
```

Reply:
That's copilot for ya. Will fix them